lib/ProvidePlugin.js

```diff
@@ -3,15 +3,18 @@
 	Author Tobias Koppers @sokra
 */
 var ModuleParserHelpers = require("./ModuleParserHelpers");
+var ModuleFilenameHelpers = require("./ModuleFilenameHelpers");
 var ConstDependency = require("./dependencies/ConstDependency");
 
 var NullFactory = require("./NullFactory");
 
-function ProvidePlugin(definitions) {
+function ProvidePlugin(definitions, options) {
```

Comment:
Could you change the signature to take only one `options` object, which contains a `definitions` property? (+Backward compatibility)